data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -110,17 +112,17 @@ protected long pos(T record) {
     return applyEqDeletes(applyPosDeletes(records));
   }
 
-  private CloseableIterable<T> applyEqDeletes(CloseableIterable<T> records) {
+  private List<Predicate<T>> applyEqDeletes() {
```

Comment:
Looks like this wasn't renamed. It doesn't apply equality deletes, so I'd rather use a more descriptive name, like `buildEqDeletePredicates`.